packages/synapse-react-client/src/utils/jsonschema/SchemaAnnotationUtils.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Utilities for retrieving annotation fields (e.g. "title", "description") from JSON Schemas
```

Comment:
The proper term for the "non-validation" fields in a JSON Schema is "annotations", so the word is a bit overloaded. This file has nothing to do with Synapse annotations.

Reply:
Not confusing at all ;)
I notice that the annotate api in hyperjump are under "experimental" packages